solr/core/src/test-files/solr/collection1/conf/solrconfig-memory-circuitbreaker.xml

```diff
@@ -82,8 +82,14 @@
 
     <useCircuitBreakers>true</useCircuitBreakers>
```

Comment:
I hate typing too much :)  and overly verbose names are often easier to mistype.

What do you think about this (note the plural `circuitBreakers` - we have more than one now!):
```
<circuitBreakers enabled="true">
 <cpuBreaker enabled="true" threshold="75"/>
 <memBreaker enabled="true" threshold="75"/>
...
</circuitBreakers>
```